### ISSUE TITLE
Display fractional scaling settings not saved on Ubuntu 24.04 with kernel 6.8

### ISSUE THREAD

**Issue by rmasad** [regolith-linux/sway-regolith#26]
## Description:
After updating to Ubuntu 24.04, which includes kernel version 6.8, my display scaling settings are not saved. The system defaults to 200% scaling upon every reboot, despite setting it to 150%. This issue does not occur with kernels prior to version 6.5.

## Steps to Reproduce:
- Update to Ubuntu 24.04.
- Ensure kernel version is 6.8.
- Set display scaling to 150%.
- Reboot the system.

## Expected Behavior:
The display scaling setting should remain at 150% after rebooting.

## Actual Behavior:
The display scaling reverts to 200% after every reboot.

## Additional Information:
Reverting to kernel 6.5 resolves the issue. Any additional information or logs that could help in diagnosing the problem.

Thank you for all the effort everyone at Regolith puts in. I hope that these reports serve to improve this system that many of us love ❤️

**Comment by SoumyaRanjanPatnaik** [comment]
Does reloading your configuration using `mod+shift+r` or `mod+shift+c` apply the fractional scale?

**Comment by rmasad** [comment]
Yes, it apply the fractional scale reloading (mod+shift+r)

**Comment by rmasad** [comment]
Yes, it solves it. Do I keep the issue open for a solution for other users or close it?
